### DISCUSSION TITLE
Olá do Brasil!

### DISCUSSION THREAD

**Discussion by eduardodarocha** [gittogethers/octogatosconf22#14] [Where in the world?]
Olá galera.

**Comment by teixtarcio** [comment]
Opa, e aí! Tudo bem?

**Comment by carvalhodanielg** [comment]
Opa

**Comment by pachicodes** [comment]
Coé a boa?

**Comment by caiotracera** [comment]
Opa, fala pessoal!

**Comment by lucalves** [comment]
Fala, turma o/

**Comment by minhadona** [comment]
Opaaa hahahaha  🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷 🇧🇷  marcando presença

**Comment by ew3g** [comment]
Salve galera

**Comment by ValchanOficial** [comment]
Olá o/

**Comment by chriscorrales** [comment]
Eae galera!

**Comment by Marujosp** [comment]
Olá pessoal ; )

**Comment by wedneyyuri** [comment]
Bom evento para todos!

**Comment by matbiit** [comment]
Olá, galera!!!! Nos vemos em breve :D

**Comment by naftalima** [comment]
Olá 😄

**Comment by fabiohcnobre** [comment]
Ola e vai ter alguem de Minas !

**Comment by GiovanaNp1** [comment]
Olá Pessoal!!! quem está animado para a Octogatos 2022??? Vamos se conectando!!

Meu instagram é [https://www.instagram.com/giovananp/](https://www.instagram.com/giovananp/)
E meu Linkedin é [https://www.linkedin.com/in/giovana-do-nascimento/](https://www.linkedin.com/in/giovana-do-nascimento/)
Tambem. recentemente Fiz meu site pessoal, ainda em construção mas ja fala sobre algumas coisas que produzo e organizo 
[Meu site](https://giovana-nascimento.netlify.app/)

Muito animada para o evento 💚💙🌼

<img width="400px" src="https://user-images.githubusercontent.com/34607779/189547409-694024d4-ead3-4244-b597-530074d666a3.gif">

**Reply by fabiohcnobre** [reply]
Que legal, estamos todos animados para Octogatos que vai acontecer!!!!
Parabens pelo trabalho 

Ate
